### DISCUSSION TITLE
Multi-wallet support

### DISCUSSION THREAD

**Discussion by Nam-notamonkey** [pwillworth/dfkreport#12] [General]
Hi there! First of all, thank you @pwillworth for all the work that you put on this tool, so far! Among other things, I'm really grateful for the addition of the Koinly universal export format and the possibility to see other Harmony transactions unrelated to DFK. The main thing that prevent me to use the tool, at the moment, is the fact that it's designed for people using only one wallet whereas (I suppose like many other people) I've been using several, moving heroes, jewels and other assets around very often... Any ETA on when you'll be able to offer multi-wallet support?

**Comment by pwillworth** [comment]
Hey, thanks for the recognition, been putting a lot of work into expanding to cover more use cases.  Unfortunately I can't offer you an ETA on multi-wallet support at this time.  All I can say is that I see it as very high on the list of things to hit on next, but at the same time it's going to be kind of a big lift because it will take some high level redesign of how the site works.  I hope that in the mean time running wallets separately and exporting the transactions and combining is at least a little help.

**Comment by pwillworth** [comment]
@Nam-notamonkey Been a while, but hopefully just in time for the new tax season, Multi-wallet support has just been added to Lila's Ledger
